config/routes.rb

```diff
@@ -13,6 +13,7 @@
     post    'signin'   => 'sessions#create',          as: :user_session
     delete  'signout'  => 'sessions#destroy',         as: :destroy_user_session
     get     'logout'   => 'sessions#destroy'
+    post    'users/unsubscribe'   => 'registrations#unsubscribe',          as: :user_unsubscribe
```

Comment:
Not sure if this is the right path to put this. I'll move it if there is a better place for it.